tests/ci/merge_pr.py

```diff
@@ -125,7 +132,11 @@ def is_approved(self, team: List[NamedUser]) -> bool:
                 return False
             return True
 
-        logging.info("The PR #%s is not approved", self.pr.number)
+        logging.info(
+            "The PR #%s is not approved by any of %s team member",
```

Comment:
why not f-string?

Reply:
Logging forces using lazy formatting everywhere

Reply:
Here's the description of why lazy formatting should be used https://stackoverflow.com/a/66993833/3706827